docs/data_collection.rst

```diff
@@ -0,0 +1,99 @@
+Data Collection
+===============
+
+This document describes how snippets data is collected, stored and used.
+
+Snippets follows Mozilla's established best practices for data collection
+and storage of personally identifiable information.
+
+Data Collection: Impressions
+----------------------------
+
+Immpression data is sampled at a rate of ~10%. For a tracked impression the following
+information is sent to the stats server.
+
+.. code-block:: json
+
+    snippet_name
+    locale
+    country
+    metric
+    campaign
+
+
+This information is sent via ajax request to https://snippets-stats.mozilla.org/ . Metric for the above
+will always be `impression`.
+
+
+Data Collection: Clicks & Events
+--------------------------------
+
+Snippets may contain a link to a Mozilla destination. When they do the link may be constructed
```

Comment:
bsmedberg specifically mentioned that we don't need to mention this, and I think we shouldn't, as this particular tracking is not covered under the same privacy policy as snippets itself is and is confusing to include IMO.